app/src/examples/AnimatedStyleUpdateExample.tsx

```diff
@@ -1,8 +1,7 @@
 import Animated, {
   useSharedValue,
-  withTiming,
   useAnimatedStyle,
-  Easing,
+  withSpring,
```

Comment:
Can we please revert all the changes to this file. It was meant to be the most basic example of using animations inside uAS. We link to it in bunch of places. If we want, we can swap out timing with spring but don't make it any more sophisticated than that (i.e. don't add more components etc)